locales/en-US.json

```diff
@@ -2,6 +2,8 @@
   "alphabetically": "Alphabetically",
   "badRequest": "Bad request. Try again",
   "browserNotSupported": "Your browser is not supported for playing this file.",
+  "changeServer": "Change Server",
```

Comment:
```suggestion
  "changeServer": "Change server",
```

We're not doing title casing anywhere else :yum:

Reply:
🤦‍♂️ I do this wrong too often 😝